### PR TITLE
Update/correct serial passthru info

### PR DESCRIPTION
**Pull request by Hwurzburg**
addresses #6273

### CODE REVIEW

File: common/source/docs/common-serial-passthrough.rst [hunk 1]
```diff
@@ -20,19 +16,18 @@ Setup
...
 
 .. figure:: ../../../images/serial-passthrough-gps-baud.png
    :target: ../_images/serial-passthrough-gps-baud.png
    :width: 300px
 
 - The default for :ref:`SERIAL_PASS1<SERIAL_PASS1>` (the first serial port to be interconnected) defaults to SERIAL0, the primary USB port.
-- Set :ref:`SERIAL_PASSTIMO <SERIAL_PASSTIMO>` to a length of time (in seconds) that gives you enough time to connect with the sensor's configuration software.  30 to 60 seconds is a good choice
+- Set :ref:`SERIAL_PASSTIMO <SERIAL_PASSTIMO>` timeout to a length of time (in seconds) that gives you enough time to connect with the sensor's configuration software. 30 to 60 seconds is a good choice but setting it to zero will remain in passthrough until reboot occurs.
 - Set :ref:`SERIAL_PASS2 <SERIAL_PASS2>` to the number of the serial port connected to the sensor.  I.e. "2" if the sensor is connected to Telem2/Serial2.
 - Press the "Disconnect" button on the ground station but leave the USB cable from the PC to the autopilot connected.
```
Comment (brad112358) [inline common/source/docs/common-serial-passthrough.rst:29]: Some GCS (QGC for example) don't have an obvious "Disconnect" button.  It's probably best to advise the user to close the GCS application to ensure the port is available for use by the configuration software or tool.

**Comment by brad112358** [comment]
I'm guessing that setting SERIALx_PROTOCOL to 0 ("None") didn't originally disable the serial port at the hardware/driver level, but just prevented it's use by the higher layers.  I ran into a case just today where this capability is still required and where the serial port locking is not good enough.  

I have an AIO flight controller with a built-in serial ELRS receiver and I was attempting to use passthrough to flash a custom firmware image using esptool.  This requires putting the receiver into bootloader mode by holding down the boot button when plugging in the USB which powers it up.  But, most of the time I couldn't get it to flash after setting up passthrough.  

It turns out that the bootloader seems to get confused by the MAVLink messages which Ardupilot was sending before I could use QGC to setup passthrough and it becomes unresponsive.  In the past, when the MLRS receiver was external to the FC, I have arranged to power the Rx only when the battery is connected and connect the battery while holding down the boot button only after starting passthrough which avoids this problem. But this solution is not available for the integrated receiver.

The workaround I found today is to set SERIALx_PROTOCOL to 28 ("Scripting") which seems a safe setting since I have no scripts so nothing would be using UART.write() to output on this port.  This completely solved the problem, but is less than ideal from a documentation point of view.  I think the ideal solution is to introduce a "unused" or "passthrough only" setting for SERIALx_PROTOCOL which ensures no output prior to enabling passthrough without disabling the serial port at the hardware/driver level.

Until this can be discussed with developers and a better solution implemented, I propose documenting the possible problem and a workaround here.  "Scripting" was the first thing I tried.  Is there a better existing setting for SERIALx_PROTOCOL which never outputs anything on the serial port.

My current workaround is documented in the Flashing section [here:](https://github.com/brad112358/mLRS-docu/blob/esp_passthrough_proto/docs/ELRS_RECEIVERS.md)

File: common/source/docs/common-serial-passthrough.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-serial-passthrough.rst:29]: done

**Comment by Hwurzburg** [comment]
I have opened a feature enhancement request #28105 for a change to protocol =0 to cover this...but don't hold your breath, the devs capable of doing this are swamped...thanks for the inputs...

**Pull request merged**